requirements/edx/coverage.txt

```diff
@@ -5,7 +5,7 @@
 #    make upgrade
 #
 coverage==5.1             # via -r requirements/edx/coverage.in
-diff-cover==2.6.1         # via -r requirements/edx/coverage.in
+diff-cover==3.0.0         # via -r requirements/edx/coverage.in
```

Comment:
@nedbat  This version adds a JSON report; do you think that could be useful for us? https://github.com/Bachmann1234/diff_cover/pull/142